README.md

```diff
@@ -18,25 +17,56 @@ var favoriteCharacter = "Jon Snow"
-* How do we then create a variable? It would be helpful to the student if there was a playground file along with this readme where the student can see the code and code along to what it is you're doing.
-* I like challenging the student a little here after you've shown them how to create a variable. In the playground file, have comments as follows:
-```Swift
-// Create another variable named favoriteLannister with the value "Tyrion Lannister"
-// Create another variable named favoriteMovie with the value of your choosing (whichever your favorite movie is).
-// Create another variable that represents your favorite color. What do you think the name of the variable should be?  
-```  
-  
-* Below each comment, the student should type out the code. I want them to begin building a habit where they read the code presented to them AND then type it out in a playground file, not copy/pasting anything.
-* Teaching them that they can indeed change the value of a variable if they want. Teach them in a way where you first phrase it as a question.. "What if we decide after a certain episode that "Jon Snow" is no longer our favorite Stark? Can we change our favoriteStark variable?
-* Similiar to how we discuss variables above, take the same approach with constants. 
-* Have the student see what it looks like when they try to change the value of a constant.
-* Make them try to change the name of a constant in the playground file.
-* Wrap up the difference between let and var
+## Variables
+
+_Variables_ are the fundamental building block of any program. A variable associates a _name_ with a _value_. That value can be almost anything: a number, a string (or sequence of characters), or even an object (which you will learn about in a future lesson). For instance, the following is a variable declaration in Swift:
+
+```
+var favoriteCharacter = "Jon Snow"
+```
+
+In the above example, a variable is declared using the `var` keyword. The name of the variable is `favoriteCharacter`. This variable has the value "Jon Snow". "Jon Snow" is a _string_, or sequence of characters enclosed in double quotation marks.
+
+You can refer to this variable using its name. While the variable's value is initially "Jon Snow", it can change over time. For example, you could change it to "Tyrion Lannister":
+
+```
+favoriteCharacter = "Tyrion Lannister"
+```
+
+From here on out, `favoriteCharacter` will have the value "Tyrion Lannister", instead of "Jon Snow".
+
+In Swift, every variable has a _type_. The type tells you the kind of value that variable refers to. In the above example, the type of the variable is a `String`. Other types include `Int` and `Float`. You will learn about many more of Swift's data types as you work through these lessons. For now, don't worry much about the type; just be aware that every variable has a type.
```

Comment:
Swift always chooses `Double` (rather than `Float) when inferring the type of floating-point numbers. Because of this, I'm slightly inclined to change where you reference `Float` here to `Double`. But in that they are both floating-point types it might be better for them to first see the word `Float` and understand what that means before throwing them into seeing the word `Double`. As I type this out, I'm now more inclined to keep it .What are your thoughts?

As per Apple,
>  In situations where either type would be appropriate, `Double` is preferred.